rpc-jobs/RPC-Artifact-Build.yml

```diff
@@ -138,3 +138,93 @@
           }} // pubcloud slave
         }} // if
       }} // cit node
+
+- job:
+    name: JJB-artifacts-apt
+    node: ArtifactBuilder2
+    concurrent: false
+    build-discarder:
+      daysToKeep: 30
+    triggers:
+      - timed: "H 4 * * *"
+    parameters:
+      - pr-params
+      - string:
+          name: ghprbTargetBranch
+          default: "{branch}"
+          description: |
+            Target branch - the branch to be tested (sha1 param) will
+            be rebased against this. Also overridden by github pull request builder plugin.
+      - string:
+          name: RPC_REPO
+          default: "https://github.com/rcbops/rpc-openstack"
+          description: "RPC REPO URL"
+      - string:
+          name: RPC_REPO_BRANCH
+          default: "artifacts-14.0"
+          description: "RPC REPO Branch"
+      - string:
+          name: ANSIBLE_DEBUG
+          default: "0"
+          description: "Enables/disables the ansible debug log"
+      - string:
+          name: ANSIBLE_VERBOSITY
+          default: "-v"
+          description: "Change the verbosity level of ansible playbook runs"
+      - string:
+          name: RPC_ARTIFACTS_FOLDER
+          default: "/var/www/artifacts"
+          description: "Folder to store artifacts (private)"
+      - string:
+          name: RPC_ARTIFACTS_PUBLIC_FOLDER
+          default: "/var/www/repo"
+          description: "Folder to store artifacts (public)"
+      - string:
+          name: DOWNLOAD_FIRST
+          default: "YES"
+          description: "Set this to NO if you don't want to fetch existing rpc mirror data -- corrupted data, or want to start fresh."
+      - string:
+          name: RECREATE_SNAPSHOTS
+          default: "NO"
+          description: "Set this to YES to destroy existing snapshot for the current artifact version"
+      - string:
+          name: PUBLISH_SNAPSHOTS
+          default: "YES"
+          description: "Set this to NO if you don't want to publish the snapshots into local folder."
+      - string:
+          name: PUSH_TO_MIRROR
+          default: "YES"
+          description: "Set this to NO if you don't want to push the published local folder, the public gpg key, and the current db to the rpc mirror."
+    wrappers:
+      - ansicolor
+      - timestamps
+      - credentials-binding:
+        - text:
+            credential-id: RPC_REPO_IP
```

Comment:
I have a card open to make sure they get added to the password sync tool, but you'll need to add them to passwordsafe as I have no idea what the unencrypted values are.